### ISSUE TITLE
Создание ивентов

### ISSUE THREAD

**Issue by vaimon** [gisss009/nash_SLON#3]
Мне кажется создание ивента в профиле - это фича отдельная от его редактирования. Возможно, стоит разделить?

**Comment by VadimProda** [comment]
Да, это хорошая идея, эти части нужно разделить. Думаю, можно переместить иконку создания ивентов рядом с иконкой редактирования профиля
![photo_2024-12-11_22-45-28](https://github.com/user-attachments/assets/22dd63c7-4cdb-45f1-b317-fbc144b11d75)

**Comment by VadimProda** [comment]
Исправлено в коммите 01fc446bc8bc65e8ed1d5a778e68b440b4ed5e75
